index.js

```diff
@@ -262,8 +262,10 @@ alexa.session = function(session) {
     this.details = {
```

Comment:
Technically speaking, since the hierarchy of the details object would match the session object, wouldn't it be easier to reduce the code to `this.details = session;`?